index.js

```diff
@@ -34,19 +65,67 @@ const colors = [ 'red' , 'white', 'black' , 'green' , 'orange'];
 //And display the results in the elements with the 'id = volume' and 'id = area' respectively
 //hint: the volumen of a sphere is ((4/3) × π × r^3) and the surface area is (4 × π × r^2)
 
+// declare the variables that is going to use in function
+const radiusField = document.getElementById('radius');
+const volumeField = document.getElementById('volume');
+const areaField = document.getElementById('area');
 
-const calculate_sphere = () =>{
+const calculate_sphere = () => {
 
+    // input the radius value 
+    let radius = radiusField.value;
+
+    // calculate the volume and area value
+    let volume = ((4/3) * Math.PI * (radius^3));
+    let area = (4 * Math.PI * (radius^2));
+    
+    // providing the results of calculation 
+    volumeField.value = volume;
+    areaField.value = area;
+
+    // debugging purpose
+    console.log(volume);
+    console.log(area);
+
+    // prevent from default browser settings
+    return false
  }
 
-window.onload = document.getElementById('MyForm').onsubmit = calculate_sphere; // this execute the volume_sphere function each time the calculate (submit) button is clicked
+document.getElementById('MyForm').onsubmit = calculate_sphere;
 
+// this execute the volume_sphere function each time the calculate (submit) button is clicked
 
 
 
 //------------------------Question 4 ---------------------------
 //Now in this Exercise we want you to create 3 buttons wich clicks actions are going to hide the respective question above with their aswer and if they are hidden then the click option should be display the question with their respective aswer
 // append the 3 buttons to the element with the 'id = a-4' as childrens
 
-
-//resolve // QUESTION 4 here
+// create variables
+const quest1 = document.getElementById('a-1');
+const quest2 = document.getElementById('a-2');
+const quest3 = document.getElementById('a-3');
+const a4Div = document.getElementById('a-4');
```

Comment:
little thing but in this questions requirements is hiding the questions with their answers. the code only hides answers not questions.